### PR TITLE
Initial removal of embezzlers

### PR DESCRIPTION
**Pull request by pstalcup**

### CODE REVIEW

File: packages/garbo/src/config.ts [hunk 1]
```diff
@@ -176,7 +196,7 @@ You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
         embezzlerMultiplier: Args.number({
           setting: "garbo_embezzlerMultiplier",
           help: "The amount we multiply our valueOfAdventure by when estimating marginal Embezzler profit. (Default 2.5)",
```
Comment (Rinn) [inline packages/garbo/src/config.ts:198]:
```suggestion
          help: "The amount we multiply our valueOfAdventure by when estimating marginal Embezzler profit. (Default 1.5)",
```

**Review by Rinn — APPROVED** [review]

**Comment by blarosen95** [comment]
Thanks for the critical fix! Is the next sprint's plan to allow users to opt out of copy fights? I feel like that would prevent another Embezzler-like outage.

**Comment by Malibu-Stacey** [comment]
@blarosen95 
> Thanks for the critical fix! Is the next sprint's plan to allow users to opt out of copy fights? I feel like that would prevent another Embezzler-like outage.

Sprint? **SPRINT?** What the actual fuck do you think this is?
This is a script maintained by people in their spare time. If you want to start paying the team they'll start adding epics & triaging the backlog & all that other Agile bullshit.

Also I think you misunderstand the entire point of this script if you it to "opt out of copy fights".

**Comment by blarosen95** [comment]
@Malibu-Stacey plenty of open source devs work in sprints for self organization. You're just a raging bitch 🙏. I wasn't making demands, merely a suggestion on what reads like someone's first PR in their life.

Guess what happened the day after this merged? It broke again because it wasn't directly disable-able. GFY.

**Pull request merged**